ui/packages/catalog-ui-search/src/main/webapp/js/widgets/openlayers.polygon.js

```diff
@@ -75,7 +75,7 @@ Draw.PolygonView = Marionette.View.extend({
   coordsToLineString: function(rawCoords) {
     const coords = []
     const setArr = _.uniq(rawCoords)
-    if (setArr.length < 3) {
+    if (setArr.length < 2) {
```

Comment:
I think polygons need at least 3 points.  Are we using the polygon drawing to do linestrings?

Reply:
ya, if we allow two points then the polygon drawing can also render the line strings